skribenten-web/frontend/src/Brevredigering/LetterEditor/actions/common.ts

```diff
@@ -165,6 +164,23 @@ export function mergeLiteralsIfPossible<T extends Identifiable>(first: Draft<T>,
   }
 }
 
+/**
+ * Splits literal text at given offset by updating 'editedText' of the given literal and returning a new literal.
+ *
+ * @param literal the literal to update
+ * @param offset the offset to split at
+ * @returns a new literal
+ */
+export function splitLiteralAtOffset(literal: Draft<LiteralValue>, offset: number): LiteralValue {
```

Comment:
For offset 0 vil vel denne gi originalteksten to gongar, men er ikkje sikkert det er ei reell problemstilling?

Reply:
Den burde oppdatere `literal` med tom streng, og flytte hele strengen over i LiteralValue som returneres.